### DISCUSSION TITLE
58

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#766] [Comments]
http://jojoldu.tistory.com/58

**Comment by jojoldu** [comment]
작성자: 눈팅이

좋은정보 감사합니다.

**Comment by jojoldu** [comment]
작성자: lsgl

좋은 정보 감사합니다.

**Comment by jojoldu** [comment]
작성자: 하비

꿀 정보 감사합니다

**Comment by jojoldu** [comment]
작성자: seungdols

좋은 정보 감사합니다.

**Comment by djpibo** [comment]
좋은 정보 감사합니다.

try catch문을 if문처럼 사용하는 건 성능을 고려했을 때 효과적이지 못하군요
